pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_common.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"fmt"
+
+	commonpb "github.com/google/tink/go/proto/common_go_proto"
+)
+
+// GetCurveType is a utility function that converts a string EC curve name into an EC curve proto type
+func GetCurveType(curve string) (commonpb.EllipticCurveType, error) {
+	switch curve {
+	case "secp256r1", "NIST_P256", "P-256", "EllipticCurveType_NIST_P256":
```

Comment:
also need 25519

Reply:
that's ed curve, not elliptic curve, will deal with this when we add chacha key support

Reply:
will be done as part of #1657